src/main/java/org/java_websocket/util/NamedThreadFactory.java

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
Could you please use the license provided in https://github.com/TooTallNate/Java-WebSocket/blob/master/LICENSE

Reply:
Done